pkg/certificate/stores.go

```diff
@@ -0,0 +1,30 @@
+package certificate
+
+import (
+	"math/rand"
+	"time"
+)
+
+const (
+	secretLength = 32
+	charset      = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+)
+
+func GetSecret() string {
+	seededRand := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
The `math/rand` module is not designed to make cryptographically secure numbers. Use the `crypto/rand` module instead, see how to do it in `crypto/encrypt_test.go`.

Also consider making a 256 bit secret and then base64 encoding it instead, for simplicity and higher entropy.

Reply:
Fixed!